src/pages/patientView/mutation/column/CohortColumnFormatter.tsx

```diff
@@ -5,35 +5,93 @@ import {IColumnFormatterData} from "../../../../shared/components/enhancedReactT
 import Tooltip from 'rc-tooltip';
 import 'rc-tooltip/assets/bootstrap_white.css';
 import {MutationTableRowData} from "../../../../shared/components/mutationTable/IMutationTableProps";
-
+import {MutSigData, VariantCountData} from "../../PatientViewPage";
+import {Mutation} from "../../../../shared/api/CBioPortalAPI";
+import {VariantCount} from "../../../../shared/api/CBioPortalAPIInternal";
 
 export default class CohortColumnFormatter {
 
     public static renderFunction(data:IColumnFormatterData<MutationTableRowData>, columnProps:any) {
-        const qValue:number = CohortColumnFormatter.getQValue(data, columnProps.data);
+        const mutSigQValue:number|null = CohortColumnFormatter.getMutSigQValue(data, columnProps.mutSigData);
+        const variantCountData = CohortColumnFormatter.getVariantCountData(data, (columnProps.variantCountData as VariantCountData));
+        const freqViz = CohortColumnFormatter.makeCohortFrequencyViz(data, (columnProps.variantCountData as VariantCountData))
         return (
-            <Td key={data.name} column={data.name}>
-                <If condition={qValue !== null}>
-                    {CohortColumnFormatter.makeMutSigIcon(qValue)};
-                </If>
+            <Td key={data.name} column={data.name} value={variantCountData && variantCountData.numberOfSamplesWithMutationInGene}>
+                <div>
+                    <If condition={freqViz !== null}>
```

Comment:
i'm curious why this passes the ts compiler because if a variable is nullable then it should require a if statement to check for its null-ness.  this is why @adufilie thinks we should stay away from <if>, as pleasant as it is